File: Solution/1662.检查两个字符串数组是否相等/1662.检查两个字符串数组是否相等.cpp
/*
 * @Descroption: LeetCode 1662. 检查两个字符串数组是否相等
 * @Author: EmoryHuang
 * @Date: 2021-05-19 20:36:31
 * @解题思路:
 * 简单题，遍历两个数组，拼接为一个字符串
 */

class Solution {
   public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2) {
        string s1, s2;
        for (auto word : word1) s1 += word;
        for (auto word : word2) s2 += word;
        return s1 == s2;
    }
};

File: Solution/1662.检查两个字符串数组是否相等/1662.检查两个字符串数组是否相等.py
'''
Descroption: LeetCode 1662. 检查两个字符串数组是否相等
Author: EmoryHuang
Date: 2022-11-01 09:10:55
Method:
简单题，直接检查两个字符串数组是否相等
'''


class Solution:

    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
        return ''.join(word1) == ''.join(word2)